src/phoenix/trace/llama_index/callback.py

```diff
@@ -528,6 +511,25 @@ def _get_end_time(event_data: CBEventData, span_events: List[SpanEvent]) -> Opti
     return _tz_naive_to_tz_aware_datetime(tz_naive_end_time)
 
 
+def _get_span_exceptions(event_data: CBEventData, start_time: datetime) -> Sequence[SpanException]:
```

Comment:
In general, output types should be explicit (and only input types need to be generic). A generic return type doesn't seem necessary here.
```suggestion
def _get_span_exceptions(event_data: CBEventData, start_time: datetime) -> List[SpanException]:
```

Reply:
I was using `Sequence` purely because it's covariant to avoid the need to cast later on. Do you think it's preferable to return the invariant `List` and `cast` later on?

Reply:
I don't see an type check error with this being `List`. 🤔

<img width="723" alt="Screenshot 2023-11-27 at 10 31 30 PM" src="https://github.com/Arize-ai/phoenix/assets/80478925/1a8de917-80b1-4361-955c-0cb1973c1a30">

Reply:
Good call, I think I only needed covariance in the input type. Addressed in 55f7a1bd.